### PR TITLE
Update hyphens.js

### PR DESCRIPTION
**Pull request by evilebottnawi**
https://developer.mozilla.org/ru/docs/Web/API/Window/scrollX
fix https://github.com/Modernizr/Modernizr/issues/1632

### CODE REVIEW

**Comment by evilebottnawi** [comment]
@patrickkettner 
Tests:
fatal: unable to access 'https://github.com/Modernizr/Modernizr.git/': Unknown SSL protocol error in connection to github.com:443

File: feature-detects/css/hyphens.js [hunk 1]
```diff
@@ -150,7 +150,14 @@ define(['Modernizr', 'prefixes', 'createElement', 'testAllProps', 'addTest'], fu
           document.body.insertBefore(div, firstChild);
           document.body.insertBefore(dummy, div);
 
-
+          var scrollX = (window.pageXOffset !== undefined)
+	          ? window.pageXOffset
```
Comment (ryanseddon) [inline feature-detects/css/hyphens.js]: The build failed to due jshint bailing out on your ternary formatting, please fix.

**Comment by evilebottnawi** [comment]
@ryanseddon @patrickkettner bump

**Comment by pzi** [comment]
@evilebottnawi You could squash those commits as most of them are just aesthetics (indenting etc).

**Comment by patrickkettner** [comment]
ping @evilebottnawi

**Comment by Schleuse** [comment]
bump

**Comment by evilebottnawi** [comment]
@patrickkettner @Schleuse  sorry for delay, an hour will do

**Comment by KuraFire** [comment]
Can someone explain what the need / purpose is of scrolling the window in a hyphens test?

**Comment by patrickkettner** [comment]
this PR resets the previous scroll position, which can be modified because of selection (I think, I honestly don't remember)

**Comment by rejas** [comment]
@KuraFire @patrickkettner this fixes #1781 (duh!) I justed tested the code, works for me. not sure about if its the right way to solve it?

**Comment by rejas** [comment]
Abother possible solution from @patrickkettner  in #1987:

> Another solution would be to do something like const el = document.querySelector("[autofocus]"); el&&el.focus(); after the test runs.
> 
> The detect itself could be updated to cache and refocus document.activeElement as well

**Comment by rejas** [comment]
Merged #2369 instead which looks cleaner. Sorry but thanks for the PR nonetheless.

**Pull request closed without merging**